autogen/paths.py

```diff
@@ -36,14 +36,17 @@ def pari_share():
     EXAMPLES::
 
         >>> from autogen.parser import pari_share
-        >>> pari_share().endswith(b'/share/pari')
+        >>> pari_share().endswith('/share/pari')
         True
     """
     from subprocess import Popen, PIPE
     if not gppath:
         raise EnvironmentError("cannot find an installation of PARI/GP: make sure that the 'gp' program is in your $PATH")
     gp = Popen([gppath, "-f", "-q"], stdin=PIPE, stdout=PIPE)
     out = gp.communicate(b"print(default(datadir))")[0]
+    # Convert out to str if needed
+    if not isinstance(out, str):
+        out = out.decode("ascii", "surrogateescape")
```

Comment:
Although I'm having second thoughts about the use of "ascii" here.  Maybe either `sys.getfilesystemencoding()` or at least utf-8?